repo/packages/C/confluent/3/marathon.json.mustache

```diff
@@ -0,0 +1,151 @@
+{
+  "id": "{{service.name}}",
+  "cpus": 1.0,
+  "mem": 1230,
+  "instances": 1,
+  "cmd": "export PATH=$PATH:$MESOS_SANDBOX/jre/bin && ./scheduler/bin/kafka-scheduler server ./scheduler/conf/scheduler.yml",
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{service.name}}",
+    "DCOS_MIGRATION_API_VERSION": "v1",
+    "DCOS_MIGRATION_API_PATH": "/v1/plan",
+    "MARATHON_SINGLE_INSTANCE_APP":"true"
+  },
+  "env": {
+    "LD_LIBRARY_PATH": "/opt/mesosphere/lib",
+    "FRAMEWORK_NAME": "{{service.name}}",
+    "USER": "{{service.user}}",
+    "PLACEMENT_STRATEGY": "{{service.placement_strategy}}",
+    "PHASE_STRATEGY": "{{service.phase_strategy}}",
+    "BROKER_COUNT": "{{brokers.count}}",
+    "BROKER_CPUS": "{{brokers.cpus}}",
+    "BROKER_MEM": "{{brokers.mem}}",
+    "BROKER_DISK": "{{brokers.disk}}",

```

Comment:
For disk type configuration, insert the following to ensure it's passed through from config.json:

```
"DISK_TYPE": "{{brokers.disk_type}}"
```

(see https://github.com/mesosphere/universe-1.7/blob/version-2.x/repo/packages/K/kafka/15/marathon.json.mustache#L23 )